src/modules/brand/components/BurnTokenDialog/BurnTokenDialog.tsx

```diff
@@ -0,0 +1,138 @@
+import { Box, Dialog, Grid, IconButton, Typography } from '@material-ui/core';
+import classNames from 'classnames';
+import { Button } from 'modules/uiKit/Button';
+import { NftType } from 'modules/createNFT/actions/createNft';
+import { InputField } from 'modules/form/components/InputField';
+import { t } from 'modules/i18n/utils/intl';
+import { useCallback } from 'react';
+import { Field, Form, FormRenderProps } from 'react-final-form';
+import { FormErrors } from 'modules/form/utils/FormErrors';
+import { AngleDownIcon } from 'modules/common/components/Icons/AngleDownIcon';
+import { AngleUpIcon } from 'modules/common/components/Icons/AngleUpIcon';
+import { CloseIcon } from 'modules/common/components/Icons/CloseIcon';
+import { useBurnTokenDialogStyles } from './useBurnTokenDialogStyles';
+import { useMemo } from 'react';
+
+const MIN_QUANTITY = 1;
+
+export interface IBurnFormValues {
+  quantity: number;
+}
+
+export const BurnTokenDialog = ({
+  loading,
+  isOpen,
+  standard,
+  maxQuantity,
+  onClose,
+  onSubmit,
+}: {
+  loading: boolean;
+  isOpen: boolean;
+  standard: NftType;
+  maxQuantity?: number;
+  onClose: () => void;
+  onSubmit: (value: IBurnFormValues, form: any, callback: any) => void;
+}) => {
```

Comment:
Better to move the typing into a named interface `IBurnTokenDialogProps` like other copmonents in the project.

Reply:
```suggestion
interface IBurnTokenDialogProps {
  loading: boolean;
  isOpen: boolean;
  standard: NftType;
  maxQuantity?: number;
  onClose: () => void;
  onSubmit: (value: IBurnFormValues, form: any, callback: any) => void;
}

export const BurnTokenDialog = ({
  loading,
  isOpen,
  standard,
  maxQuantity,
  onClose,
  onSubmit,
}: IBurnTokenDialogProps) => {
```